http/src/main/resources/jupyter/jupyter-docker-compose-gce.yaml

```diff
@@ -0,0 +1,31 @@
+# Note: we need to stay on docker-compose version 2 because version 3 doesn't support
+# configuring memory options in container mode. See discussion in:
+# https://docs.docker.com/compose/compose-file/#resources
+# https://github.com/docker/compose/issues/4513
+version: '2'
+services:
+  jupyter:
+    container_name: "${JUPYTER_SERVER_NAME}"
+    image: "${JUPYTER_DOCKER_IMAGE}"
+    # Override entrypoint with a placeholder to keep the container running indefinitely.
+    # The cluster init script will run some scripts as root and then start pyspark as
+    # jupyter-user via docker exec.
+    entrypoint: "tail -f /dev/null"
+    network_mode: host
+    volumes:
```

Comment:
Created this spin off with Hadoop/Spark related volumes removed. Didn't make the Scala changes since we'll refactor ClusterHelper, etc.